pages/auth/signup.tsx

```diff
@@ -1,7 +1,7 @@
 import { SubmitHandler, useForm } from "react-hook-form";
 
 import useAuthContext from "@App/lib/auth/AuthContext";
-import styles from "@App/styles/LoginPage.module.scss";
+//import styles from "@App/styles/LoginPage.module.scss";
```

Comment:
we should delete these unused imports and class names below

Reply:
okay will do